backend/manager/modules/dal/src/main/resources/bundles/ExecutionMessages.properties

```diff
@@ -148,6 +148,7 @@ job.RemoveUnregisteredVm=Remove Unregistered VM ${VmName}
 job.SwitchMasterStorageDomain=Switching master storage domain from ${OldMaster} to ${NewMaster}
 job.ConvertDisk=Converting disk ${DiskName} to ${AllocationPolicy}/${DiskFormat}
 job.HybridBackup=Backing up VM ${VM}
+job.ReduceImage=Reducing the actual size of Disk ${DiskAlias}
```

Comment:
Not sure `Disk` should start with uppercase.

Reply:
that's the convention in the execution job messages, see where `DiskAlias` is used above